packages/gatsby-cli/src/structured-errors/error-schema.ts

```diff
@@ -20,6 +20,7 @@ export const errorSchema: Joi.ObjectSchema<IStructuredError> = Joi.object().keys
         })
       )
       .allow(null),
+    category: Joi.string().valid([`USER`, `SYSTEM`]),
```

Comment:
What would we chose as category for "Wordpress host is down right now and we're getting errors retrieving data" ? Maybe we can have a `THIRD_PARTY` category for that.

Reply:
good point, I would have marked that as user

Reply:
THIRD_PARTY added